server/src/configs/sequelize.js

```diff
@@ -0,0 +1,36 @@
+require("dotenv").config();
+const { Sequelize } = require("sequelize");
+
+try {
+  const sequelize = new Sequelize(
+    process.env.DATABASE_NAME,
+    process.env.DATABASE_USERNAME,
+    process.env.DATABASE_PASSWORD,
+    {
+      host: process.env.DATABASE_HOST,
+      dialect: "mysql",
+      dialectOptions: {
+        ssl: {
+          rejectUnauthorized: true,
+        },
+      },
+      define: {
+        defaultScope: {
+          attributes: { exclude: ["createdAt", "updatedAt"] },
+        },
+        timestamps: false,
+        freezeTableName: true,
+      },
+    },
+  );
+
+  sequelize.authenticate().catch(() => {
+    console.error("Error connecting to database");
```

Comment:
Linter error here due to console.log. Should be able to just use Pino to handle that

Reply:
Linter error fixed :D